sunkit_image/utils/NoiseLevel.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+from scipy.stats import gamma
+from scipy.ndimage import correlate
+from skimage.util import view_as_windows
+
+
+class NoiseLevelEstimation:
+    def __init__(self, img, patchsize=7, decim=0, conf=1-1E-6, itr=3):
+        """
+        NoiseLevel estimates noise level of input single noisy image.
+
+        Input parameters:
```

Comment:
This should use numpydoc styling, i.e.
```
Parameters
----------
```

same as the other section headings :)